### ISSUE TITLE
missing files? PrepSpatial.r PlotMap.r PlotMapMulti.r

### ISSUE THREAD

**Issue by remi-daigle** [OHI-Science/can-atl#2]
I think the repo is missing these files?

**Comment by jules32** [comment]
Yes, I've just commented them out temporarily. We are in the process of adding them to ohicore and thus updating [calculate_scores.r#L25-L390](https://github.com/OHI-Science/can-atl/blob/master/region2017/calculate_scores.r#L25-L390). 

We also need to think through this setwd() thing more because ohicore expects you to be within the scenario folder (setwd('region2017')) so things are a bit grumpy now. 

But you don't need to be making figures yet so you can proceed!

**Comment by remi-daigle** [comment]
ok, cool!

I personally thing the `setwd()` thing is worth fixing (it drives me a little crazy). Since you guys are updating `ohicore` anyway, what about if you implement something like this:

```
testfunction <- function(filename,wd=NULL){
  print(paste0(wd,filename))
}

testfunction("layers.csv")

testfunction("layers.csv","region2017/")
```
By adding `wd` with the default value of `NULL` as an argument to all the relevant functions. You allow anal people like me to avoid `setwd()` since I can then simply supply my directory info. Additionally, it shouldn't break old code since not providing a `wd` means it takes on a `NULL` value and doesn't really affect anything. Just note that this has to be done with `paste0()` not `file.path()` since the latter will return `character(0)` if `wd=NULL`
